style.css

```diff
@@ -0,0 +1,281 @@
+/* Add your CSS here */
+/* Dont' forget to link this file to your HTML in the <head> */
+
+/* header */
+header {
+  position: sticky;
+  background-color: white;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+.header-box {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1400px;
+  margin: 0 auto;
+}
+
+.header-nav-menu {
+  display: flex;
+}
+.header-nav-menu li {
+  padding: 20px;
+  list-style: none;
+  font-size: 0.9rem;
+  font-family: "Quicksand", sans-serif;
+}
+
+.about-me{
+    color:#0150fd ; 
+}
+
+.header-name {
+  font-size: 1.6rem;
+  font-family: "Poppins", sans-serif;
+}
+.header-title {
+  font-size: 1rem;
+  font-family: "Quicksand", sans-serif;
+}
+.box1 {
+  background-color: blue;
+  height: 15px;
+  width: 15px;
+}
+
+.box1-name {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
+.background1 {
+  background: #e6dacd;
+  width: 40%;
+  height: 90vh;
+  position: absolute;
+  z-index: -1;
+}
+.main-content {
+  display: flex;
+  height: 90vh;
+  gap: 1rem;
+  align-items: center;
+  justify-content: center;
+}
+.main-box-container {
+  background-color: #f4ece6;
+  box-shadow: rgba(0, 0, 0, 0.19) 0px 10px 20px, rgba(0, 0, 0, 0.23) 0px 6px 6px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 1rem;
+  gap: 2.5rem;
+}
+img {
+  width: 50%;
+  height: auto;
+  border-radius: 50%;
+}
+.main-name {
+  font-size: 1.6rem;
+  font-family: "Poppins", sans-serif;
+  text-align: center;
+}
+
+hr.blue-line {
+  background-color: #0150fd;
+  width: 19%;
+  height: 2px;
+}
+
+.main-title {
+  font-size: 1.1rem;
+  letter-spacing: 0.5rem;
+  font-family: "Quicksand", sans-serif;
+}
+
+.contact {
+  background-color: white;
+  align-items: center;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  height: 3rem;
```

Comment:
Typically we avoid setting explicit heights in layout. Could you think about using gap and padding to create this component instead?